LandSandBoat/server
# [Modules] Simplify HNM system logic

**_I affirm:_**

## What does this pull request do?

Simplifies logic of both hnm system modules and makes sure they work.

Unrelated to the actual issue in Issue #3412 but it was mentioned.

## Steps to test these changes

Enable ONE of them.
Kill a Land King
Make sure server variables get set/updated when it despawns
